### PR TITLE
remove everything from the global scope

### PR DESCRIPTION
**Pull request by czardoz**
Remove all injected objects from globals, and instead use variable references to them.

### CODE REVIEW

File: lib/uvm/vm-execute.js [hunk 1]
```diff
@@ -7,35 +7,51 @@
...
+        __done = universe.__done,
+        __async = universe.__async,
+        __delete = universe.__delete,
+        __require = universe.__require,
+        __masked = universe.__masked,
+        requires = universe.requires,
+        errors = universe.errors,
+        replacements;
+
+    __delete && __delete.length && __delete.forEach(function(key) {
```
Comment (czardoz) [inline lib/uvm/vm-execute.js]: Who will delete `__delete`! :D

Comment (shamasis) [inline lib/uvm/vm-execute.js]: lol!

File: lib/uvm/vm-execute.js [hunk 2]
```diff
@@ -46,7 +62,7 @@
         }
         catch (e) {
             libraries[lib.object] = undefined;
-            universe.on.exception(e); // We'll try to run without this library
+            on.exception(e); // We'll try to run without this library
```
Comment (shamasis) [inline lib/uvm/vm-execute.js]: where is `on` defined?

Comment (shamasis) [inline lib/uvm/vm-execute.js]: found it

**Pull request merged**